apps/docs/integrations/adonis-v5.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Adonis V5'
+sidebarTitle: 'Adonis V5'
+description: 'Add React Email to any Adonis v5 with TypeScript project in minutes.'
+'og:image': 'https://react.email/static/covers/react-email.png'
+---
+
+import NextSteps from '../snippets/next-steps.mdx';
+
+## 1. Install
+
+Install package from your command line.
+
+<CodeGroup>
+
+```sh npm
+npm install @react-email/render -E
+```
+
+```sh yarn
+yarn add @react-email/render -E
+```
+
+</CodeGroup>
+
+I created a folder called emails to store the JSX files
```

Comment:
We should avoid using "I". I think "Create a folder..." sounds better.

Reply:
Resolved